File: app/src/main/java/br/com/douglas/speaktous/model/JWTPostagem.java
package br.com.douglas.speaktous.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class JWTPostagem {

    @SerializedName("users")
    @Expose

    private int status;
    private String message;
    private Postagem postagem;

    public JWTPostagem() {
    }

    public int getStatus() {
        return status;
    }

    public void setStatus(int status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public Postagem getPostagem() {
        return postagem;
    }

    public void setPostagem(Postagem postagem) {
        this.postagem = postagem;
    }
}

File: app/src/main/java/br/com/douglas/speaktous/model/JWTPostagem.kt
package br.com.douglas.speaktous.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class JWTPostagem {

    @SerializedName("users")
    @Expose
    var status: Int = 0
    var message: String? = null
    var postagem: Postagem? = null
}
